.github/workflows/reusable-pytest.yml

```diff
@@ -68,10 +71,15 @@ jobs:
       - name: Setup python for tox
         uses: actions/setup-python@v5
         with:
-          python-version: 3.9
+          python-version: 3.12
```

Comment:
```suggestion
          python-version: "3.12"
```

Reply:
Actually, I think this isn't needed anymore, since I changed to to be installed via `uv tool`. Let's check that first.